js/client-chat.js

```diff
@@ -1738,7 +1738,9 @@
 			} else if (mayNotify && this.id.substr(0, 5) === 'help-') {
 				this.notifyOnce("Help message from " + name, "\"" + message + "\"", 'pm');
 			} else if (mayNotify && name !== '~') { // |c:|~| prefixes a system message
-				this.subtleNotifyOnce();
+				if (!Dex.prefs('chatmute' + this.title)) {
```

Comment:
Thanks for the feedback; I've updated my implementation to use a table instead of cluttering the top level with arbitrary text (oops):
![Screen Shot 2021-01-07 at 7 22 40 PM](https://user-images.githubusercontent.com/49170515/103959464-bfe09500-511d-11eb-90b7-fb5ce080a884.png)

 Also changed the parameter to roomid rather than title.